packages/connect/src/core/index.ts

```diff
@@ -565,7 +562,7 @@ export const onCall = async (message: CoreMessage) => {
             } catch (error) {
                 // catch wrong pin error
                 // PinMatrixAck returns { code: "Failure_PinInvalid", message: "PIN invalid"}
-                if (error.message === 'PIN invalid' && PIN_TRIES < MAX_PIN_TRIES) {
+                if (error.message.includes('PIN invalid') && PIN_TRIES < MAX_PIN_TRIES) {
```

Comment:
Ongoing discussion with @mroz22 as to how to get this bug fix into trezor/connect

Reply:
no problem with this change

Reply:
just do it in separate commit with proper commit message please

Reply:
@mroz22 Yeah sure, separated into it's own commit here [eb094a2](https://github.com/trezor/trezor-suite/pull/9576/commits/eb094a2674c1eb8894654188231e4f6391afbe98)